index.html

```diff
@@ -681,18 +681,18 @@
 </p>
                       <!--script(src=dir+'/controller.js')-->
                       <script>app.controller('BsdateCtrl', function($scope) {
-  $scope.user = {
-    dob: new Date(1984, 4, 15)
-  };
+	$scope.user = {
```

Comment:
Please don't change all the indentation without making any actual code changes.  This just increases the delta count for diff.